src/redux/incidents/actions.js

```diff
@@ -101,6 +105,11 @@ export const filterIncidentsByTeam = (teamIds = []) => ({
   teamIds,
 });
 
+export const filterIncidentsByEscalationPolicy = (escalationPolicyIds = []) => ({
+  type: FILTER_INCIDENTS_LIST_BY_SERVICE,
```

Comment:
Needs to be `FILTER_INCIDENTS_LIST_BY_ESCALATION_POLICY`

Reply:
```suggestion
  type: FILTER_INCIDENTS_LIST_BY_ESCALATION_POLICY,
```